shell/platform/tizen/flutter_tizen_engine.cc

```diff
@@ -38,8 +38,8 @@ static DeviceProfile GetDeviceProfile() {
   return DeviceProfile::kUnknown;
 }
 
-FlutterTizenEngine::FlutterTizenEngine(bool headed)
-    : device_profile(GetDeviceProfile()) {
+FlutterTizenEngine::FlutterTizenEngine(bool headed, void* win)
+    : device_profile(GetDeviceProfile()), custom_window_(win) {
```

Comment:
What if the user uses a custom window with properties we don't want or invalid?

Reply:
If you don't want it, just set it to null. Then the window will be created internally as before.
At invalid case, Currently there is no countermeasures for that situation . So maybe the app is having a problem.

Reply:
> At invalid case, Currently there is no countermeasures for that situation . So maybe the app is having a problem.

Do you have any plan for this?

Reply:
I'm going to try to find a way to check the validity, but I have idea that ultimately the parameters transmitted from the outside should be checked from the outside.

Reply:
Ok, It's still in the development stage, so we'll continue to discuss it next time, and let's move on with this PR.